src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer {
```

Comment:
Raw type usage in machine producer

```suggestion
public class ExcavatorProducer implements MachineProducer{
```

Reply:
Java code style says that the space on the contrary should be